deployment/ccip/changeset/solana_state.go

```diff
@@ -8,18 +8,25 @@ import (
 	"github.com/gagliardetto/solana-go"
 
 	"github.com/smartcontractkit/chainlink/deployment"
+	commontypes "github.com/smartcontractkit/chainlink/deployment/common/types"
 )
 
 var (
-	LinkToken     deployment.ContractType = "LinkToken"
-	SolCcipRouter deployment.ContractType = "SolCcipRouter"
+	SolCcipRouter         deployment.ContractType = "SolCcipRouter"
+	SolAddressLookupTable deployment.ContractType = "SolAddressLookupTable"
+	SolTokenPool          deployment.ContractType = "SolTokenPool"
+	SolCcipReceiver       deployment.ContractType = "SolCcipReceiver"
 )
 
 // SolChainState holds a Go binding for all the currently deployed CCIP programs
 // on a chain. If a binding is nil, it means here is no such contract on the chain.
 type SolCCIPChainState struct {
-	LinkToken     solana.PublicKey
-	SolCcipRouter solana.PublicKey
+	LinkToken             solana.PublicKey
+	SolCcipRouter         solana.PublicKey
+	SolTokenPool          solana.PublicKey
+	Timelock              solana.PublicKey
+	SolAddressLookupTable solana.PublicKey // for chain writer
+	SolCcipReceiver       solana.PublicKey // for tests only
```

Comment:
`Receiver` (name used on EVM side too), etc. the type itself is already called `SolCCIPChainState`